mods/cnc/sequences/misc.yaml

```diff
@@ -33,56 +35,51 @@ smoke_m:
 laserfire:
 	idle:veh-hit3
 		Length: *
+		ZOffset: 511
 
 dragon:
 	idle:
 		Facings: 32
+		ZOffset: 1023
 
 smokey:
 	idle:
 		Length: *
+		ZOffset: 1023
 
 bomb:
 	idle:
 		Length: *
+		ZOffset: 1023
 
 missile:
 	idle:
 		Facings: 32
+		ZOffset: 1023
 
 patriot:
 	idle:
 		Facings: 32
+		ZOffset: 1023
 
 explosion:
-	nuke_explosion: atomsfx
+	Defaults:
```

Comment:
Something is off here, shouldn't this be already done by #10099 ([here](https://github.com/OpenRA/OpenRA/pull/10099/files#diff-41204c0cc5225c86612ce7ec415ae9f1L58))?